en/06_Project_Governance/06_Supported_Modules.md

```diff
@@ -0,0 +1,114 @@
+[silverstripe/recipe-services](https://github.com/silverstripe/recipe-services) | 2
+[silverstripe/registry](https://packagist.org/packages/silverstripe/registry) | 3
+[silverstripe/restfulserver](https://packagist.org/packages/silverstripe/restfulserver) | 3
+[silverstripe/securityreport](https://packagist.org/packages/silverstripe/securityreport) | 3
+[silverstripe/segment-field](https://packagist.org/packages/silverstripe/segment-field) | 3
+[silverstripe/sharedraftcontent](https://packagist.org/packages/silverstripe/sharedraftcontent) | 3
+[silverstripe/sitewidecontent-report](https://packagist.org/packages/silverstripe/sitewidecontent-report) | 4
+[silverstripe/spamprotection](https://packagist.org/packages/silverstripe/spamprotection) | 4
+[silverstripe/staticpublishqueue](https://packagist.org/packages/silverstripe/staticpublishqueue) | 6
+[silverstripe/subsites](https://packagist.org/packages/silverstripe/subsites) | 3
+[silverstripe/tagfield](https://packagist.org/packages/silverstripe/tagfield) | 3
+[silverstripe/taxonomy](https://packagist.org/packages/silverstripe/taxonomy) | 3
+[silverstripe/textextraction](https://packagist.org/packages/silverstripe/textextraction) | 4
+[silverstripe/totp-authenticator](https://packagist.org/packages/silverstripe/totp-authenticator) | 5
+[silverstripe/userforms](https://packagist.org/packages/silverstripe/userforms) | 6
+[silverstripe/versionfeed](https://packagist.org/packages/silverstripe/versionfeed) | 3
+[silverstripe/webauthn-authenticator](https://packagist.org/packages/silverstripe/webauthn-authenticator) | 5
+[symbiote/silverstripe-advancedworkflow](https://packagist.org/packages/symbiote/silverstripe-advancedworkflow) | 6
+[symbiote/silverstripe-gridfieldextensions](https://packagist.org/packages/symbiote/silverstripe-gridfieldextensions) | 4
+[symbiote/silverstripe-multivaluefield](https://packagist.org/packages/symbiote/silverstripe-multivaluefield) | 6
+[symbiote/silverstripe-queuedjobs](https://packagist.org/packages/symbiote/silverstripe-queuedjobs) | 5
+[tractorcow/silverstripe-fluent](https://packagist.org/packages/tractorcow/silverstripe-fluent) | 7
+
+## Supported NPM packages
+
+The following two NPM packages are also supported because they are required to build the UI of Silverstripe CMS 5:
+- [@silverstripe/webpack-config](https://www.npmjs.com/package/@silverstripe/webpack-config)
+- [@silverstripe/eslint-config](https://www.npmjs.com/package/@silverstripe/eslint-config)
+
+You'll only need to interact with them if you are customising the Silverstripe CMS UI.
+
+## Other non-supported modules
+
+They are other modules hosted under the _silverstripe_ Packagist handle. These modules are maintained on a best effort basis. They do not go through regular regression testing. Their APIs may be more fluid than supported modules. They maybe more experimental or may not receive the same level of care as supported modules.
```

Comment:
This completely ignores the fact that some are maintained by members of bespoke teams, and _could_ be maintained by community members. I think we should add something about that in here - especially since the changelog explicitly asks people to let us know if they want to maintain some of these.

Maybe we should have a way to identify modules which have maintainers (list them in the readme perhaps?)

Reply:
I don't think we need to discuss the internal structure of Silverstripe in the official docs. I don't think there's anything in the previous module that wouldn't apply to a bespoke module. e.g.: TerraFormers could decide that they don't give a toss about the embargo module tomorrow, or they could tag a new major release, or they could could decide to fix a security issue without issuing a CVE.

Reply:
It's not about the internal structure of Silverstripe, we're inviting community members to maintain some of these modules, and frankly anyone not in the cms squad is effectively "the community" from the standpoint of our documentation. So saying anything in the silverstripe github org which isn't a supported module "do not go through regular regression testing" basically is saying that community maintainers of those repos aren't doing much... which we can't really say. I'll recommend a specific change in a separate comment.

Reply:
I've added the "no guarantee of regression testing" bit as suggested.